models/sessions/scratch/snowplow_unified_session_screen_summary_metrics.sql

```diff
@@ -0,0 +1,41 @@
+{#
+Copyright (c) 2024-present Snowplow Analytics Ltd. All rights reserved.
+This program is licensed to you under the Snowplow Personal and Academic License Version 1.0,
+and you may not use this file except in compliance with the Snowplow Personal and Academic License Version 1.0.
+You may obtain a copy of the Snowplow Personal and Academic License Version 1.0 at https://docs.snowplow.io/personal-and-academic-license-1.0/
+#}
+
+{{
+  config(
+    sql_header=snowplow_utils.set_query_tag(var('snowplow__query_tag', 'snowplow_dbt'))
+  )
+}}
+
```

Comment:
It doesn't _really_ matter as we make sure the columns are populated with nulls, but should we only enable this table if the context is enabled in the project?

Reply:
Makes sense, I have added `enabled` flag here to match the variable.